### PR TITLE
Change generate_tables from genrule to .bzl rule.

### PR DESCRIPTION
**Pull request by tomlu**
genrules do not work on Windows without a msys dependency.

The tool had to be changed as well, since ctx.actions.run does not
support piping stdin and stdout to files. I couldn't find any
other dependencies on the tool in this repository, but the visibility
is public so if any external dependencies have decided to use
this they will break.

### CODE REVIEW

File: generatetables/generate_tables.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ distributed under the License is distributed on an "AS IS" BASIS,
 package main
 
 import (
+    "flag"
```
Comment (pmbethe09) [inline generatetables/generate_tables.go:20]: gofmt ?

File: lang/defs.bzl [hunk 2]
```diff
@@ -0,0 +1,24 @@
+def _generate_tables_impl(ctx):
+    args = ctx.actions.args()
+    args.add("--input", ctx.file.src)
```
Comment (pmbethe09) [inline lang/defs.bzl:3]: opt: can use single '-' in go flags '-input'

File: generatetables/generate_tables.go (hunk 1 above)
Comment (tomlu) [reply generatetables/generate_tables.go]: Done.

File: lang/defs.bzl (hunk 2 above)
Comment (tomlu) [reply lang/defs.bzl]: Done.

**Pull request merged**